osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/DefaultDeviceResponseServiceTest.java

```diff
@@ -0,0 +1,139 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.opensmartgridplatform.adapter.domain.core.infra.jms.ws.WebServiceResponseMessageSender;
+import org.opensmartgridplatform.shared.exceptionhandling.OsgpException;
+import org.opensmartgridplatform.shared.exceptionhandling.TechnicalException;
+import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessageResultType;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+
+
+import org.junit.jupiter.api.Test;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class DefaultDeviceResponseServiceTest {
+
+
+    @Mock
+    private WebServiceResponseMessageSender webServiceResponseMessageSender;
+    
+    @InjectMocks
+    private DefaultDeviceResponseService defaultDeviceResponseService;
+    
+    @Test
+    public void testDefaultDeviceResponseWithNotOkTypeAndException() {
+    	//Arrange
+    	CorrelationIds ids = new CorrelationIds("orginazationTestId", "deviceIdTest", "correlationUid");
```

Comment:
new CorrelationIds("orginazationTestId", "deviceIdTest", "correlationUid") is used in more tests, refactor it into a constant CORRELATION_IDS